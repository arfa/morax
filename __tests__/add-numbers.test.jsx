describe('Test Adding Numbers', () => {
  it('Expect 1 + 2 to be 3', () => {
    expect(1 + 2).toBe(3)
    expect(3).toMatchSnapshot()
  })
})
