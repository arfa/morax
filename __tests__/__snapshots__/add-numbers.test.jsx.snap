// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Adding Numbers Expect 1 + 2 to be 3 1`] = `3`;
